fe/fe-core/src/main/java/com/starrocks/transaction/DatabaseTransactionMgr.java

```diff
@@ -424,7 +424,11 @@ public void prepareTransaction(long transactionId,
                 txnSpan.setAttribute("num_partition", numPartitions);
                 unprotectedCommitSpan.end();
                 // after state transform
-                transactionState.afterStateTransform(TransactionStatus.PREPARED, txnOperated, null);
+                try {
+                    transactionState.afterStateTransform(TransactionStatus.PREPARED, txnOperated, null);
```

Comment:
the memory object is modified, but the editlog is not written here. wondering if the exception is ignored here and then write the editlog in line 433, will it cause any inconsistency? will it be a proper error handling?